### ISSUE TITLE
srccd: `Objects/plot_object.py` raises TypeError

### ISSUE THREAD

**Issue by Andrew95496** [Andrew95496/cobra#9]
```
Traceback (most recent call last):
  File "/Users/drewskikatana/Documents/Programming/cobra/multiscrape.py", line 184, in <module>
    plot.create_view()
  File "/Users/drewskikatana/Documents/Programming/cobra/Objects/plot_object.py", line 23, in create_view
    graph = self._create_graph(data)
  File "/Users/drewskikatana/Documents/Programming/cobra/Objects/plot_object.py", line 15, in _create_graph
    plt.hist(data, 100)
  File "/opt/homebrew/lib/python3.9/site-packages/matplotlib/pyplot.py", line 2590, in hist
    return gca().hist(
  File "/opt/homebrew/lib/python3.9/site-packages/matplotlib/__init__.py", line 1412, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/matplotlib/axes/_axes.py", line 6611, in hist
    xmin = min(xmin, np.nanmin(xi))
  File "<__array_function__ internals>", line 180, in nanmin
  File "/opt/homebrew/lib/python3.9/site-packages/numpy/lib/nanfunctions.py", line 351, in nanmin
    res = np.amin(a, axis=axis, out=out, **kwargs)
  File "<__array_function__ internals>", line 180, in amin
  File "/opt/homebrew/lib/python3.9/site-packages/numpy/core/fromnumeric.py", line 2916, in amin
    return _wrapreduction(a, np.minimum, 'min', axis, None, out,
  File "/opt/homebrew/lib/python3.9/site-packages/numpy/core/fromnumeric.py", line 86, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
TypeError: '<=' not supported between instances of 'float' and 'str'
```

**Comment by Andrew95496** [comment]
#Possible Solution

- https://stackoverflow.com/questions/46406720/labelencoder-typeerror-not-supported-between-instances-of-float-and-str

**Comment by Andrew95496** [comment]
Issue fixed: New issue raised #10
